scripts/cmake-presets/ci-ubuntu-rocm.json

```diff
@@ -1,11 +1,11 @@
 {
-  "version": 3,
-  "cmakeMinimumRequired": {"major": 3, "minor": 21, "patch": 0},
+  "version": 6,
+  "cmakeMinimumRequired": {"major": 3, "minor": 23, "patch": 0},
   "configurePresets": [
     {
       "name": "base",
-      "displayName": "CentOS7/rocm-docker default options",
```

Comment:
Do we have test on CentOS8 and/or EL9?

Reply:
Rocky9 (the new CUDA docker image) is a very close relative to both of those right? I think it's the spiritual successor to CentOS.

Reply:
> Rocky9 (the new CUDA docker image) is a very close relative to both of those right?

it is a detail but why did you go with Rocky9 then? (AlmaLinux/EL9 are used in HEP, I am not sure Rocky is)

Reply:
Availability of CUDA docker images I think? I looked at the Linux family tree to try to make a determination

Reply:
Fair enough.  Thanks.